### PR TITLE
Phone ext provider grid

### PR DESCRIPTION
**Pull request by jrabolli**

### CODE REVIEW

File: db/migrate/20190524192535_add_language_to_providers.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddLanguageToProviders < ActiveRecord::Migration[5.2]
+  def change
+    add_column :providers, :language, :string
```
Comment (mocerinoj) [inline db/migrate/20190524192535_add_language_to_providers.rb]: Can the provider speak multiple languages?

Comment (jrabolli) [reply db/migrate/20190524192535_add_language_to_providers.rb]: yes - mostly English, Spanish, or both.  I am told some creole as well at a couple facilities.  not sure I handled this totally correctly but after trying different approaches wanted to get it out there without breaking partial UI

Comment (mocerinoj) [reply db/migrate/20190524192535_add_language_to_providers.rb]: so how is this going to handle "both"? "multi"? This seems like it should be a jsonb array long term

Comment (jrabolli) [reply db/migrate/20190524192535_add_language_to_providers.rb]: I actually thought of doing it like you handled services for facility but then wondered If it was too much.  Is that the direction you see?

File: app/views/facilities/_providers.html.erb [hunk 2]
```diff
@@ -2,7 +2,12 @@
   <thead>
     <tr>
       <th>Name</th>
+<<<<<<< HEAD
```
Comment (mocerinoj) [inline app/views/facilities/_providers.html.erb]: This needs to be fixed

**Pull request merged**